app/helpers/icon_helper.rb

```diff
@@ -6,12 +6,10 @@ module IconHelper
   CHAR_ICON_FAKE = 'char-access-icon char-access-fake pointer'.freeze
 
   def icon_tag(icon, **args)
-    return '' if icon.nil?
```

Comment:
Seems this is necessary when doing character replaces - do we need to remove these guards we have here + below as part of this MR?